docs/detections/detection-engine-intro.asciidoc

```diff
@@ -63,19 +63,20 @@ To make sure you can access Detections and manage rules, see
 
 [float]
 [[cold-tier-detections]]
-== Compatibility with cold tier nodes
+== Compatibility with cold and frozen tier nodes
 
-Cold tier is a {ref}/data-tiers.html[data tier] that holds time series data that is accessed only occasionally. In {stack} version >=7.11.0, {elastic-sec} supports cold tier data for the following {es} indices:
+Cold and frozen {ref}/data-tiers.html[data tiers] hold time series data that is only accessed occasionally. In {stack} version >=7.11.0, {elastic-sec} supports cold tier data for the following {es} indices:
```

Comment:
I agree with adding mention of frozen tiers for clarity, but it doesn't seem like this line things much clearer. If I'm understanding correctly, we say:

1. X supports cold tier
2. Y and Z do not support cold or frozen tier

It seems like we should be explicit on whether X (which is the list on lines 70-72) supports frozen tier or not.

Reply:
```suggestion
Cold and frozen {ref}/data-tiers.html[data tiers] hold time series data that is only accessed occasionally. In {stack} version >=7.11.0, {elastic-sec} supports cold but not frozen tier data for the following {es} indices:
```

Does this look good?

Reply:
It addresses my issue,  but I defer to @marshallmain to confirm that this statement is correct

Reply:
This is roughly correct but I think we on the dev/product side need to come up with a more precise concept of what exactly we support because separating things by index pattern isn't sufficient granularity IMO. We do support frozen tier data existing in index patterns queried by detection rules as long as the frozen data is not in the time range queried by the rule. In general for the use cases listed here it's important that frozen tier data isn't queried regularly, i.e. that the time range on the query excludes frozen tier data (or a _tier filter? I can't remember if we tested to confirm that optimization works or not), but cold/frozen tier data is allowed to live in those index patterns.

So for detection rules, we might say "frozen tier data must not be in the time range queried by the rule or it must be explicitly excluded with a _tier filter". For new terms rules, we should also specify that frozen tier data can't be in the (interval + lookback) OR the history window. Similarly for the "data sources selector on various pages", the frozen data only needs to be excluded from the query.

Reply:
I don't think we need to hold up this PR to rework the concepts here though.

Reply:
@marshallmain I'm tracking your feedback in a private docs issue so we don't forget to revisit these docs in the future.

https://github.com/elastic/security-docs-internal/issues/27